Automattic/jetpack
# AI Assistant: Disable Breve when AI Assistant block is disabled

Fixes #

## Proposed changes:

* Changes the Breve availability check

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to a site with a free plan
* Check that Breve is not loaded
* Add a plan
* Check that Breve is enabled
* Hide the Jetpack AI Assistant block
* Check that Breve is disabled
* Unhide the block
* Check that Breve is re-enabled without a need to reload
